bots/push-rewrite

```diff
@@ -54,6 +61,10 @@ def main():
     api = github.GitHub(repo=opts.repo)
     old_statuses = api.statuses(remote)
 
+    # add [no-test] to title to prevent tests bring triggered
+    pull = find_pr_with_sha(api, remote)
+    api.post("pulls/{0}".format(pull["number"]), "{0} [no-test]".format(pull["title"]))
```

Comment:
As this works on an existing PR by definition, couldn't this now use the label? This would avoid github notification email noise.

Reply:
oh yea, then that's preferred